spec/requests/citizens/additional_accounts_spec.rb

```diff
@@ -16,6 +16,19 @@
     end
   end
 
+  context 'when an applicant revisits the page to change their answer' do
+    before do
+      application.update!(has_offline_accounts: true)
+      application.update!(state: 'use_ccms')
+    end
+
+    it 'checks that offline account is reset to nil' do
+      get citizens_additional_accounts_path
+      expect(application.reload.has_offline_accounts).to be_nil
+      expect(application.reload.state).to eq 'provider_submitted'
+    end
```

Comment:
You don't need the second reload here - it was reloaded after the controller action in line 27.